packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -60,7 +60,66 @@ export class WaferMap extends FoundationElement {
         values: []
     };
 
```

Comment:
It would be nice if we could have one or more unit tests that verify this new functionality but it's challenging because nothing about the public state of the component is changing. Here are the options I can think of, most of them not great:
1. write a test that uses a Jasmine spy to verify that `queueRender` gets called after setting one (or all) of these properties. I believe spies can intercept calls to TypeScript private functions, though this would be tying the test to implementation details.
2. expose some public state just for testing (e.g. `renderQueued`). This isn't great because it adds public API that clients don't need.
3. don't write any tests now and instead rely on Chromatic tests that will be added in the PR that implements rendering

I think I'd vote for 1 but welcome other ideas or opinions.

Reply:
Yeah, for 1 I was thinking that the same spy mechanism you pseudocoded would work (since the function you spy on is passed as a string I guess it wouldn't fail compile despite the function being private). But thinking about it more I like the `@internal` approach better. That signals both a) this is technically public so if you change its name you should search for references and b) it's not intended to be used by clients.